### PR TITLE
[Feature] Add ticket booking functionality

### PR DESCRIPTION
**Pull request by cahogan**
This PR will allow a user ("Sweetwest employee") to book tickets on behalf of a customer.

https://github.com/user-attachments/assets/b4e8214f-5838-4a32-8c21-9aa720ff3f75

### CODE REVIEW

**Comment by CoolNamesAllTaken** [comment]
❤️

File: core/static/core/favicon.svg
Comment (CoolNamesAllTaken) [inline core/static/core/favicon.svg:1]: glowed up a lil

File: core/views/ticket.py [hunk 1]
```diff
@@ -17,3 +18,38 @@ def ticket(request, ticket_id: int = None):
...
+                group, number = get_boarding_position(current_bookings)
+                ticket = Ticket.objects.create(
+                    flight=flight,
+                    first_name=request.POST.get("customer_name"),
+                    costume=request.POST.get("customer_costume"),
+                    tsa_precheck=checkbox,
+                    boarding_group=group,
+                    boarding_position=number,
+                )
+                ticket.save()
```
Comment (CoolNamesAllTaken) [inline core/views/ticket.py:48]: Does this have the possibility of assigning multiple people the same boarding number?

Comment (cahogan) [reply core/views/ticket.py:48]: yep! To be addressed in a subsequent PR :)

**Pull request merged**